src/main/webapp/js/instructorCourseDetails.js

```diff
@@ -44,27 +48,35 @@ function submitFormAjax() {
     });
 }
 
-/**
- * Functions to trigger registration key sending to a specific student in the
- * course.
- * Currently no confirmation dialog is shown.
- */
-function toggleSendRegistrationKey() {
-    return confirm('Usually, there is no need to use this feature because TEAMMATES '
-                   + 'sends an automatic invite to students at the opening time of each'
-                   + ' session. Send a join request anyway?');
+function attachEventToRemindStudentsButton() {
```

Comment:
Try to write stateless functions. i.e. the function takes all it needs as parameters and returns the result. That way, the function becomes easier to test. The testability and test coverage of our current js code is not that good.